portal-impl/test/unit/com/liferay/portal/servlet/ComboServletTest.java

```diff
@@ -273,6 +273,39 @@ public void testServiceWithPortletIdAndProxyAndContext() throws Exception {
 			_pluginServletContext);
 	}
 
+	@Test
+	public void testTooManyComboServletRequests() throws Exception {
```

Comment:
:thinking: Isn't this test requesting exactly 100 files, which is the default maximum allowed? How can it return BAD REQUEST, then?

Reply:
Oh, you're right 🤦